src/Altinn.App.Core/Features/Validation/ValidationAppSI.cs

```diff
@@ -232,7 +232,8 @@ public async Task<List<ValidationIssue>> ValidateDataElement(Instance instance,
         private List<ValidationIssue> MapModelStateToIssueList(
             ModelStateDictionary modelState,
             Instance instance,
-            string dataElementId)
+            string dataElementId,
+            object data)
```

Comment:
Change method signature to only require what is needed in the method
```suggestion
            Type data)
```